spec/support/form_helpers.rb

```diff
@@ -227,6 +268,16 @@ def fill_in_activity_form(
     expect(page).to have_content gdi
     expect(page).to have_content flow
     expect(page).to have_content t("activity.aid_type.#{aid_type.downcase}")
+    if level == "project" || level == "third_party_project"
+      expect(page).to have_content policy_marker_gender
```

Comment:
These probably need `within` blocks as multiple policy markers could contain the same value.

This is currently looking for "Not assessed" somewhere on the page, which would still be true even if the summary page was missing some of the markers.

Reply:
That's true. I'll change that